### PR TITLE
Enable selinux enforcing

### PR DESCRIPTION
**Pull request by zimmerry**

### CODE REVIEW

**Comment by zimmerry** [comment]
[Enable SELinux Enforcing](https://trello.com/c/WlALaNKE/930-enable-selinux-enforcing)

File: recipes/default.rb [hunk 1]
```diff
@@ -16,6 +16,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if platform_family?('rhel')
+  include_recipe 'osl-selinux'
+end
```
Comment (ramereth) [inline recipes/default.rb:21]: This is a cleaner way to do it. Please fix it globally.
```suggestion
include_recipe 'osl-selinux' if platform_family?('rhel')
```

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Please do a minor bump on `~`

**Comment by zimmerry** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.5.0.  Have a nice day!

**Pull request merged**